src/hermes/commands/deposit/invenio.py

```diff
@@ -69,16 +78,24 @@ def deposit(click_ctx: click.Context, ctx: CodeMetaContext):
 
     _log = logging.getLogger("cli.deposit.invenio")
 
+    invenio_config = config.get("deposit").get("invenio", {})
     invenio_path = ContextPath.parse("deposit.invenio")
     invenio_ctx = ctx[invenio_path]
 
     if not click_ctx.params["auth_token"]:
         raise DepositionUnauthorizedError("No auth token given for deposition platform")
-    click_ctx.session.headers["Authorization"] = f"Bearer {click_ctx.params['auth_token']}"
+
+    http_client = click_ctx.obj.get("http_client")
+    http_client.headers["Authorization"] = f"Bearer {click_ctx.params['auth_token']}"
```

Comment:
Clever idea hiding the complexity of requests here. However, this might have unexprected side effects (e.g., if you want multiple deposits or other steps use the http_client, too).

Reply:
Good point. In the worst case, we could leak the authentication token to some other site which could then, based on the hermes user agent, guess where to apply this token (some site targeted by hermes). I'll come up with something different.

Reply:
I made the requests.Session local in 5236f9a84ae2d791c901ff1ae1554142e6236198